src/entity/posts/utils.ts

```diff
@@ -414,14 +414,16 @@ export const createSharePost = async ({
       },
     } as DeepPartial<SharePost>);
 
-    const vordrStatus = await checkWithVordr(
-      {
-        id: createdPost.id,
-        content: createdPost.title || undefined,
-        type: VordrFilterType.Post,
-      },
-      { con, userId, req: ctx?.req },
-    );
+    const vordrStatus = ctx
+      ? await checkWithVordr(
+          {
+            id: createdPost.id,
+            content: createdPost.title || undefined,
+            type: VordrFilterType.Post,
+          },
+          ctx,
```

Comment:
I don't think this is right, `userId` before did not come from `ctx` but from `authorId` param, not sure if it can be different user but technically it could

Reply:
It can't be. I'll check if it's easy to fix. No one can write a post on behalf of someone else, and also if they did, you still want to rate limit the logged in user